src/Explorer/Tabs/TabsBase.ts

```diff
@@ -32,6 +33,7 @@ export default class TabsBase extends WaitsForTemplateViewModel {
   public isExecutionError: ko.Observable<boolean>;
   public isExecuting: ko.Observable<boolean>;
   public pendingNotification?: ko.Observable<DataModels.Notification>;
+  public manager = ko.observable<TabsManager>(undefined);
```

Comment:
Does this need to be wrapped in observable? No problem if yes just would be nice to skip if not needed

Reply:
Thanks, I made some changes and it is no longer an Observable, nice catch.